doc/users/next_whats_new/EngFormatter-offset.rst

```diff
@@ -0,0 +1,12 @@
+ticker.EngFormatter now computes offset by default
+--------------------------------------------------
+
+:class:`matplotlib.ticker.EngFormatter` has gained the ability to show an offset text near the
+axis. With shared logic with :class:`matplotlib.ticker.ScalarFormatter`, it is capable of
+deciding whether the data qualifies having an offset and show it with an
+appropriate SI quantity prefix, and with the supplied ``unit``.
+
+To enable this new behavior, simply pass ``useOffset=True`` when you instantiate
+:class:`matplotlib.ticker.EngFormatter`. Example is available here_.
+
+.. _here: :doc:`gallery/ticks/engformatter_offset`
```

Comment:
> And underneath you can also link to the example via plot_directive to embed the image [matplotlib.org/devdocs/devel/document.html](https://matplotlib.org/devdocs/devel/document.html#include-figures-and-files)

Thanks, the link helped, and I used the examples there so probably it will work now, let's see.